clients/iceberg-views/src/main/java/org/apache/iceberg/view/BaseMetastoreViews.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.iceberg.view;
```

Comment:
Should we maintain "org.apache.iceberg.view" package in a fork of iceberg instead of Nessie ? It looks odd IMO.

Reply:
yeah I think we just want to maintain the same package name as it will have when this gets into iceberg